control_plane/attachment_service/src/http.rs

```diff
@@ -311,6 +311,17 @@ async fn handle_status(_req: Request<Body>) -> Result<Response<Body>, ApiError>
     json_response(StatusCode::OK, ())
 }
 
+/// Readiness endpoint indicates when we're done doing startup I/O (e.g. reconciling
+/// with remote pageserver nodes)
```

Comment:
Mentioning Kubernetes here would be helpful, then one knows the purpose/meaning of this API.

Reply:
Added that to the comment.